### PR TITLE
Initial addition of documentation for core actions.

### LINKED ISSUE #389: Add documentation for the core actions
The core actions (e.g. sendCommand, executeCommandLine, sendGetHttpRequest, etc.) are missing from the OH 2 docs.

https://github.com/openhab/openhab1-addons/wiki/Actions#http-actions
http://docs.openhab.org/addons/actions.html

### PR DESCRIPTION
**Pull request by rkoshak**
Closes #389

I added the documentation for the core Actions. I'm not 100% happy with the formatting but couldn't think of anything better (table?) so I stuck with how they are in the wiki.

Signed-off-by: Richard Koshak <rlkoshak@gmail.com>

### CODE REVIEW

File: addons/actions.md [hunk 1]
```diff
@@ -7,8 +7,89 @@ title: Actions
...
+Actions are predefined methods that are called from openHAB rules and scripts.
+They are automatically imported and can be used to execute openHAB-specific operations or to send commands or data to external hardware or services. There are a set of built in core Actions and optional installable Actions.
+
+## Core Actions
+
+The following Actions are a core part of openHAB and do not need to be separately installed.
+
+### Event Bus Actions
+
+* `sendCommand(String itemName, String commandString)`: Sends the given command to the spcified Item to the event bus.
```
Comment (ThomDietrich) [inline addons/actions.md]: "spcified"

File: addons/actions.md [hunk 2]
```diff
@@ -7,8 +7,89 @@ title: Actions
...
+* `logDebug(String loggername, String logText)`: Logs logText on level DEBUG to openhab.log
+* `logInfo(String loggername, String logText)`: Logs logText on level INFO to openhab.log
+* `logWarn(String loggername, String logText)`: Logs logText on level WARN to openhab.log
+* `logError(String loggername, String logText)`: Logs logText on level ERROR to openhab.log
+
+`logText` can be a compete String, constructed through concatination, or through arguments.
+
+**Complete String Example:** `logInfo("Garage", "This is a complete String")`
+**Concatination Example:** `logDebug("Lighting", "This is a string concatination:" + Light.name)`
+**Arguments Example:** `logWarn("Alarm", "The {} window has been open for {} hours!", Window.name, timeOpen)`
```
Comment (ThomDietrich) [inline addons/actions.md]: These will be joined without newlines. Either itemize them or force a break at the end of the lines with `<br />`

File: addons/actions.md [hunk 3]
```diff
@@ -7,8 +7,89 @@ title: Actions
...
+* `sendHttpPostRequest(String url)`: Sends a POST-HTTP request and returns the result as a String
+* `sendHttpPostRequest(String url, String contentType, String content)`: Sends a POST-HTTP request with the given content and returns the result as a String
+* `sendHttpDeleteRequest(String url)`: Sends a DELETE-HTTP request and returns the result as a String
+
+### Timers
+
+`createTimer(AbstractInstant instant, Procedure procedure)`: schedules a block of code to execute at a future time
+
+`instant` is usually a `DateTime` calculated using the built in variable `now`.
+`procedure` is the block of code and is defined using lambda notation (i.e. square brackets)
```
Comment (ThomDietrich) [inline addons/actions.md]: itemize these?

Comment (rkoshak) [inline addons/actions.md]: I think I've done this correctly. I made the updates and pushed the changes. I don't need to do anything else to make is show up here as part of the PR, right? 

All suggestions were implemented.

**Pull request merged**